.github/workflows/vdr-creation.yml

```diff
@@ -0,0 +1,46 @@
+name: VDR Creation
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request: # todo: do we want it to run on all PRs?
```

Comment:
You won't be able to use secrets in PR runs so it probably makes sense to remove this

Reply:
we can run in slow mode without the secrets (and there's a chance of some other PR or other github action elsewhere causing some throttling, although i've not seen this in trial runs), but getting the vdr result of a PR before merging I think is super valuable. Without the secret, the rate limit is 5 requests/30 seconds (with the secret it's 50). Experimentally, it takes <30 minutes to get the vdr generated without the secret, so I think it's worth waiting 30 mins and spot checking the vdr before actually merging a PR. In the future, we might integrate the GH actions cache to further speed up the CI.